00.prerequisites/README.md

```diff
@@ -57,9 +57,30 @@ If you have a new Linux install, just do:
 sudo apt install python3 python3-venv python3-pip python3-flask
 ```
 
+## Virtual Environment Setup
+
+Virtual environment ensures isolation from the global system configuration.
+Read about it here: https://docs.python-guide.org/dev/virtualenvs/
+
+I am not using virtual environment for this project.
+Feel free to use it though.
+It is especially beneficial if you intend to pack the service into a Docker container:
+
+```
+alex@latitude7490:~/Projects/RESTing-with-Flask/02.rest/farm/ > python3 -m venv venv
+alex@latitude7490:~/Projects/RESTing-with-Flask/02.rest/farm/ > source ./venv/bin/activate
+(venv) alex@latitude7490:~/Projects/RESTing-with-Flask/02.rest/farm/ > pip3 install flask requests
```

Comment:
Now that the `requirements.txt` is included:
```
pip install -r requirements.txt
```
More on this: https://note.nkmk.me/en/python-pip-install-requirements/

Reply:
I added comments to use `pip3 install -r requirements.txt` when there is requirements.txt available. 

I tried both pip3 and pip and they produced the same result. Google says Python3 uses pip3 so I used that to be consistent with the rest of the document.